test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
Do we want to allow this? Ideally I'd think it should be `b? : number`? Or is it customary to put the space before the question mark?